Assets/PickUpItem.cs

```diff
@@ -0,0 +1,10 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PickUpItem : MonoBehaviour {
+
+	// Update is called once per frame
+	public void buff (float attack_increase, bool ice_fire, int increase_fireball_number) {
```

Comment:
Use capitalize Function name.
This interface is too limited.
Maybe use some generic way?
I have finished a meteor and working on the fire Rain possibly. Try make it compatible by this interface?